cli/deepsea.py

```diff
@@ -119,80 +122,144 @@ def _validate_stage_file_exists(stage_name):
         sys.exit(1)
 
 
-def _run_show_stage_steps(stage_name, hide_state_steps, only_visible_steps, use_cache):
+def _print_stage_step(step, indent, step_order_map):
+    if isinstance(step, SaltState):
+        PP.print(PP.orange(step.sls))
+        PP.print(PP.dark_green(" ({})".format(step.desc)))
+        PP.println(PP.orange(" on"))
+        if step.on_success_deps or step.on_fail_deps:
```

Comment:
Why do we check this twice? Can't we simplify these nested if branches to something like:
```python
if step.on_success_deps:
    # success branch
elif step.on_fail_deps:
   # fail branch
else:
    PP.println()

Reply:
Not in this case because the `PP.println()` in the `else` branch can only execute if the `step.on_success_deps` evaluates to true, otherwise it should not be executed at all.